estimatefee.go

```diff
@@ -0,0 +1,434 @@
+// Copyright (c) 2013-2016 The btcsuite developers
+// Use of this source code is governed by an ISC
+// license that can be found in the LICENSE file.
+
+package main
+
+import (
+	"errors"
+	"fmt"
+	"math"
+	"math/rand"
+	"sort"
+	"sync"
+
+	"github.com/btcsuite/btcd/chaincfg/chainhash"
+	"github.com/btcsuite/btcutil"
+)
+
+const (
+	// estimateFeeDepth is the maximum number of blocks before a transaction
```

Comment:
The variable name as declared and what's referenced in the comment are mismatched. The comment should be updated to reference `estimateFeeBins` instead of `estimateFeeDepth`.